platform-tests/src/test/java/org/junit/platform/commons/util/ClasspathScannerTests.java

```diff
@@ -230,6 +238,44 @@ void scanForClassesInPackageOnlyLoadsClassesThatAreIncludedByTheClassNameFilter(
 	}
 
 	@Test
+	void scanForClassesInSpringBootPackageIsAbleToLoadClassesCorrectly() throws Exception {
+		File jarFile = createFakeSpringBootJar(ClasspathScannerTests.class);
+		ClassLoader springBootClassLoader = new ClassLoader() {
+			@Override
+			public Enumeration<URL> getResources(final String name) throws IOException {
+				final Vector<URL> resources = new Vector<>();
+				resources.add(new URL("jar:file://" + jarFile.getAbsolutePath() + "!/BOOT-INF/classes!/org/junit/platform/commons"));
+				return resources.elements();
```

Comment:
This could be shorted to

```java
URL url = ...;
return Collections.enumeration(Collections.singleton(url));
```

but again it's not critical. :)